django/db/models/query.py

```diff
@@ -645,6 +645,16 @@ def create(self, **kwargs):
         Create a new object with the given kwargs, saving it to the database
         and returning the created object.
         """
+        params = set(kwargs.keys())
```

Comment:
`.keys()` call is unnecessary
```suggestion
        params = set(kwargs)
```